guacamole/src/main/webapp/app/login/directives/login.js

```diff
@@ -188,17 +197,37 @@ angular.module('login').directive('guacLogin', [function guacLogin() {
 
                     // Clear all remaining fields that are not username fields
                     angular.forEach($scope.remainingFields, function clearEnteredValueIfPassword(field) {
-
+                        
                         // If field is not username field, delete it.
                         if (field.type !== Field.Type.USERNAME && field.name in $scope.enteredValues)
                             delete $scope.enteredValues[field.name];
-
+                        
                     });
+
```

Comment:
Not a deal-breaker, but if you can, I suggest rebasing away these whitespace changes. I'm not sure where they came from, but they don't appear related to the changes.

Reply:
> Unneeded whitespaces removed

It's partly fixed, yes, but the two instances of added whitespace above (lines 204 and 206) are still here.